src/new-apps/components/NotInstalledAppListRow/NotInstalledAppListRow.tsx

```diff
@@ -0,0 +1,118 @@
+import TableButtonWrapper from "@dashboard/components/TableButtonWrapper/TableButtonWrapper";
+import TableCellAvatar from "@dashboard/components/TableCellAvatar";
+import TableRowLink from "@dashboard/components/TableRowLink";
+import { JobStatusEnum } from "@dashboard/graphql";
+import { buttonMessages } from "@dashboard/intl";
+import { useAppListContext } from "@dashboard/new-apps/context";
+import {
+  appInstallationStatusMessages,
+  appsMessages,
+} from "@dashboard/new-apps/messages";
+import { AppInstallation } from "@dashboard/new-apps/types";
+import { CircularProgress, TableCell, Typography } from "@material-ui/core";
+import {
+  Button,
+  DeleteIcon,
+  IconButton,
+  Indicator,
+  Pill,
+  Tooltip,
+  TooltipMountWrapper,
+} from "@saleor/macaw-ui";
+import clsx from "clsx";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import AppManifestTableDisplay from "../AppManifestTableDisplay";
+import { useStyles } from "./styles";
+
+export const NotInstalledAppListRow: React.FC<AppInstallation> = props => {
+  const { appInstallation, isExternal, logo } = props;
```

Comment:
```suggestion
export const NotInstalledAppListRow: React.FC<AppInstallation> =({ appInstallation, isExternal, logo }) => {
```

Reply:
I left `props` deconstruction on purpose below because it is easier to pass `props` to `useStyles(props)` later on.